.github/workflows/docker.yml

```diff
@@ -195,7 +222,22 @@ jobs:
     ########################################
     # lumai/askem-skema-text-reading
     ########################################
-    - name: Tags for image
+    - name: Setup JDK (w/ SBT)
+      uses: actions/setup-java@v3
+      with:
+        distribution: temurin
+        java-version: 11
+        # Unfortunately, this will immediately 
+        # blow through the GH Actions cache...
+        #cache: 'sbt'
+    - name: Generate Dockerfile (TR)
+      working-directory: ./skema/text_reading/scala
+      env:
+        #APP_VERSION: ${{needs.app_version.outputs.app_version}}
+        APP_VERSION: ${{needs.app_version.outputs.commit}}
+      run: |
+        sbt "webapp/docker:stage"
```

Comment:
In the scala subproject's build.sbt I see `webapp/docker:publishLocal`.  This might produce a better result (or maybe not).  I did fiddle with some things via docker.sbt that affect the layout of the files.  As far as I recall, the result is an image without sbt and even without Scala except for the runtime library.  The docker support is a relatively new ability of sbt.